lib/rucio/core/replica.py

```diff
@@ -742,7 +742,7 @@ def _list_replicas_for_files(file_clause, state_clause, files, rse_clause, updat
 def _list_replicas(dataset_clause, file_clause, state_clause, show_pfns,
                    schemes, files, rse_clause, rse_expression, client_location, domain,
                    sign_urls, signature_lifetime, constituents, resolve_parents,
-                   updated_after, vo,
+                   updated_after, filters,
```

Comment:
Why is this argument `filters` and not `filter`?

Reply:
At line 767/766 we actually use the built in python `filter()` function, so the normal practice of naming it `filter` leads to an overwrite and the whole function falls over (like the default arguments for `{}` this another one that pylint doesn't like but normally you can get away with). There are other places we use the argument `filters`, so I just used that instead.

Reply:
In general and from now on we should use the term `filters` but it's not worth retroactively changing `filter` to `filters`.